tracing/opencensus/grpc.go

```diff
@@ -0,0 +1,155 @@
+package opencensus
+
+import (
+	"context"
+
+	"go.opencensus.io/trace"
+	"go.opencensus.io/trace/propagation"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+
+	kitgrpc "github.com/go-kit/kit/transport/grpc"
+)
+
+const propagationKey = "grpc-trace-bin"
+
+// GRPCClientTrace enables OpenCensus tracing of a Go kit gRPC transport client.
+func GRPCClientTrace(options ...TracerOption) kitgrpc.ClientOption {
+	cfg := TracerOptions{
+		sampler: trace.AlwaysSample(),
```

Comment:
By default each trace is always being sampled, perhaps let's put that as a comment in the doc right above?
```go
// By default each trace is sampled, however you can change this behavior
// by using the TracerOption WithSampler
```
or something like that